.github/ISSUE_TEMPLATE/Bug report form.yml

```diff
@@ -0,0 +1,127 @@
+name: Report a problem/bug
+title: '[Problem/Bug]: '
+description: Report a problem if something's not working.
+labels: ["bug"]
+body:
+- type: markdown
+  attributes:
+    value: 
+      Thanks for taking the time to fill out this report. Here is [an example](https://github.com/MicrosoftEdge/WebView2Feedback/issues/3648) of a good bug report that will help us in accurately identifying the issue.
+- id: what-happened
+  type: textarea
+  attributes:
+    description: Please tell us what's not working as expected. Please include callstacks and error messages if possible.
+    label: What happened?
+    placeholder: Please describe the issue you encountered. Callstacks and error messages are helpful.
+  validations:
+    required: true
+- id: importance
+  type: dropdown
+  attributes:
+     label: Importance
+     description: May you help us understand the level of importance of this ask to you?
+     multiple: false
+     options:
+       - ----Please select----
+       - Blocking. My app's basic functions cannot work.
+       - Important. My app's user experience is significantly compromised.
+       - Moderate. This issue has a certain impact on my app. 
+  validations:
+      required: true
+- id: runtime-channel
```

Comment:
Is this question needed if we get the version number below?

Reply:
I tend to keep it, as this question also serves the purpose of nudging developers to try out prerelease more. It also makes calculating easier if we want to pull out some stats.